features/old/stats/provider_side.feature

```diff
@@ -52,12 +52,16 @@ Feature: Provider stats
     When I log in as provider "foo.3scale.localhost"
     Given a product
     And a backend
+    And the following BackendApi type metrics:
+      | pasta  |
+      | Pizza  |
+    And the following metrics:
+      | Curry |
```

Comment:
This doesn't work, and it's because the new step `the following BackendApi type metric(s):` is adding the new backend to the existing service. To ensure this works. The backend must be created before the service. So you must arrange everything to:

1. Create the backend
2. Create the service
3. add the backend to that service.

These are the exact steps required to reproduce the issue as they are detailed in the [issue](https://issues.redhat.com/browse/THREESCALE-8719) description.